data/languages.yaml

```diff
@@ -108,6 +108,7 @@ en:
         - (\d+)\s*sec(s?): \1 second\2
         - noon: '12:00'
         - midnight: '00:00'
+        - (\d+)h(\d+)m?: \1:\2
```

Comment:
It is. Because translated string is passed onto dateutil parser, where time component (20h30m) is changed to `20 hour 30 minute` and is not supported by it.